### PR TITLE
Issue/#2

### PR DESCRIPTION
**Pull request by dumbturtle**
Add test function for pure(#2) and fixes issue #4.

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -22,6 +21,7 @@ flake8-string-format==0.3.0
 flake8-tidy-imports==4.1.0
 flake8-typing-imports==1.9.0
 flake8-variables-names==0.0.3
+mr-proper==0.0.5
```
Comment (Melevir) [inline requirements_dev.txt:24]: we should add `mr-proper` to requirements in setup.py also.

File: flake8_functions/checker.py [hunk 2]
```diff
@@ -127,7 +127,13 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
...
-                full_error_info = *error_info, type(self)
-                yield full_error_info
+
+            error_amount_info = self._get_arguments_amount_error(func_def, self.max_parameters_amount)
+            if error_amount_info:
+                full_error_amounnt_info = *error_amount_info, type(self)
+                yield full_error_amounnt_info
+
+            error_pure_info = check_purity_of_functions(func_def)
+            if error_pure_info:
```
Comment (Melevir) [inline flake8_functions/checker.py:137]: This is seems like copy-paste of if above (line 132). It would be nice to refactor this.

**Comment by dumbturtle** [comment]
I've finished. Please, check it.

**Comment by Melevir** [comment]
Thanks!

**Pull request merged**